### PR TITLE
Add the grammarly extension

### PR DESCRIPTION
**Pull request by JesterOrNot**
This will greatly reduce the number of needed grammar edit PRs that I along with others make. And will greatly reduce overall grammar errors from ever entering the code-base.

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -12,21 +12,23 @@ vscode:
...
     - paulmolluzzo.convert-css-in-js@1.1.3:YnjK47pXScU3DMFfQzkkOw==
     
+    - znck.grammarly@0.12.2:Tbs7TRx8olec3FCWN9TjvA==
+
 github:
   prebuilds:
     # enable for the master/default branch (defaults to true)
     master: true
-    # enable for all branches in this repo (defaults to false)
+# enable for all branches in this repo (defaults to false)
```
Comment (RDIL) [inline .gitpod.yml:21]: This is a bad YAML practice.  It should be indented.

Comment (JesterOrNot) [reply .gitpod.yml:21]: auto-formatting oops!

**Comment by nisarhassan12** [comment]
@JesterOrNot Thanks this looks exciting but I wonder How can I use this extension I tried `F1 > Grammarly: check grammar errors` but it seems to do nothing. 

Note: For testing, I have added some invalid text to the markdown file.

**Comment by JesterOrNot** [comment]
blocked by https://github.com/znck/grammarly/issues/104

**Review by jankeromnes — COMMENTED** [review]
Personally, I wouldn't feel comfortable using an extension that uploads everything I type to a third-party service. 😬 (But maybe I'm being too paranoid here.)

**Pull request closed without merging**